docs/guides/advanced/alternative-clients.md

```diff
@@ -44,3 +44,70 @@ exporter = OTLPSpanExporter(
     headers={'Authorization': 'your-write-token'},
 )
 ```
+
+
+## Example with Rust
+
+First, set up a new Cargo project:
+
+```sh
+cargo new --bin otel-example && cd otel-example
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://logfire-api.pydantic.dev
+export OTEL_EXPORTER_OTLP_HEADERS='Authorization=your-write-token'
+```
+
+Update the `Cargo.toml` and `main.rs` files with the following contents:
+
+```toml
+# Cargo.toml
+[package]
+name = "otel-example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+opentelemetry = { version = "0.24", default-features = false, features = ["trace"] }
+opentelemetry-otlp = { version = "0.17", default-features = false, features = ["trace", "http-proto", "reqwest-blocking-client", "reqwest-rustls"] }
+```
+
+```rust
+// src/main.rs
+use opentelemetry::{
+    global::ObjectSafeSpan,
+    trace::{Tracer, TracerProvider},
+};
+
+fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    // If you don't want to export environment variables, you could also just
```

Comment:
Could this configuration be done in the exporter construction like in the Python example?

Reply:
Yes, done.